2_process/src/process_and_style.R

```diff
@@ -10,7 +10,10 @@ annotate_data <- function(site_data_clean, site_filename){
   annotated_data <- left_join(site_data_clean, site_info, by = "site_no") %>% 
     select(station_name = station_nm, site_no, dateTime, water_temperature, latitude = dec_lat_va, longitude = dec_long_va)
   
-  return(annotated_data)
+  # make station name a factor
+  styled_data <- style_data(annotated_data)
```

Comment:
One fun thing is that because `style_data()` accepts a data frame as its first argument and also returns a data frame, you could just include it as part of your pipe chain above, e.g.

```r
annotated_data <- left_join(site_data_clean, site_info, by = "site_no") %>% 
    select(station_name = station_nm, site_no, dateTime, water_temperature, latitude = dec_lat_va, longitude = dec_long_va) %>%
    style_data() # make station name a factor
```